test/compiler/parsing/fixtures/watf/inf/expected.json

```diff
@@ -0,0 +1,31 @@
+{
+  "type": "Program",
+  "body": [
+    {
+      "type": "Instr",
+      "id": "const",
+      "object": "f32",
+      "args": [
+        {
+          "type": "Identifier",
+          "value": "inf"
+        }
+      ]
+    },
+    {
+      "type": "Instr",
+      "id": "const",
+      "object": "f32",
+      "args": [
+        {
+          "type": "UnaryExpression",
+          "operator": "-",
+          "argument": {
+            "type": "Identifier",
```

Comment:
Tracking it as an Identifier is a potential issue, I tracked it here https://github.com/xtuc/js-webassembly-interpreter/issues/94.